frame/lottery/src/weights.rs

```diff
@@ -167,8 +168,8 @@ impl WeightInfo for () {
 	// Storage: Lottery TicketsCount (r:1 w:1)
 	// Storage: Lottery Tickets (r:1 w:0)
 	fn on_initialize_end() -> Weight {
-		// Minimum execution time: 62_720 nanoseconds.
-		Weight::from_ref_time(63_545_000 as u64)
+		// Minimum execution time: 61_712 nanoseconds.
+		Weight::from_ref_time(62_144_000 as u64)
```

Comment:
This as u64 is still present here, should be gone in 769c2e4f3634f71a2f13914ea3c50f34eda40754

Reply:
Should be good now, thanks!